api/main.js

```diff
@@ -10,6 +11,27 @@ const routes = require('./routes');
 
 const server = express();
 
+if (process.env.ENDPOINT_COVERAGE_CAPTURE) {
+  server.use((req, res, next) => {
+    const end = res.end.bind(res);
+    res.end = (...args) => {
+      let path = req.path;
```

Comment:
alternatively, `const path = req.route ? req.route.path : req.path` to save a couple lines and avoid the reassignment (though i don't care in the least if you want to leave as-is)